.github/workflows/lint.yml

```diff
@@ -19,3 +19,5 @@ jobs:
         gem install bundler dotenv
         bundle install --jobs 4 --retry 3
         bundle exec rubocop
+    - name: Enforce EditorConfig
+      uses: zbeekman/EditorConfig-Action@28b76e718c351dd35b22c8f7216192bb9de5c21b
```

Comment:
Had to use SHA because of npm audit issue in action https://github.com/zbeekman/EditorConfig-Action/issues/35